opentelemetry-api/src/opentelemetry/metrics/__init__.py

```diff
@@ -49,160 +47,118 @@ class Meter:
-                on recording when suppling values for these labels.
-            span_context: The `SpanContext` that identifies the `Span`
-                that the metric is associated with.
-
-        Returns: A new `FloatCounter`
-        """
-
-    def create_int_counter(
-        self,
-        name: str,
-        description: str,
-        unit: str,
-        label_keys: LabelKeys,
-        span_context: SpanContext = None,
-    ) -> "IntCounter":
-        """Creates a counter type metric that contains int values.
-
-        Args:
-            name: The name of the counter.
-            description: Human readable description of the metric.
-            unit: Unit of the metric values.
-            label_keys: list of keys for the labels with dynamic values.
-                Order of the list is important as the same order MUST be used
-                on recording when suppling values for these labels.
-            span_context: The `SpanContext` that identifies the `Span`
-                that the metric is associated with.
-
-        Returns:
-            A new `IntCounter`
-        """
-
-    def create_float_gauge(
-        self,
-        name: str,
-        description: str,
-        unit: str,
-        label_keys: LabelKeys,
-        span_context: SpanContext = None,
-    ) -> "FloatGauge":
-        """Creates a gauge type metric that contains float values.
-
-        Args:
-            name: The name of the counter.
-            description: Human readable description of the metric.
-            unit: Unit of the metric values.
-            label_keys: list of keys for the labels with dynamic values.
-                Order of the list is important as the same order MUST be used
-                on recording when suppling values for these labels.
-            span_context: The `SpanContext` that identifies the `Span`
-                that the metric is associated with.
-
-        Returns:
-            A new `FloatGauge`
-        """
-
-    def create_int_gauge(
-        self,
-        name: str,
-        description: str,
-        unit: str,
-        label_keys: LabelKeys,
-        span_context: SpanContext = None,
-    ) -> "IntGauge":
-        """Creates a gauge type metric that contains int values.
-
-        Args:
-            name: The name of the counter.
-            description: Human readable description of the metric.
-            unit: Unit of the metric values.
-            label_keys: list of keys for the labels with dynamic values.
-                Order of the list is important as the same order MUST be used
-                on recording when suppling values for these labels.
-            span_context: The `SpanContext` that identifies the `Span`
-                that the metric is associated with.
-
-        Returns:
-            A new `IntGauge`
-        """
+    # TODO: RecordBatch
+
+
+class ValueType(Enum):
+    FLOAT = 0
+    INT = 1
+
+
+def create_counter(
```

Comment:
Are these static now?  I feel like it should be in some Meter object.

Reply:
Good question. The rationalization behind having the creates as a module level function is that we do not want to depend on the user having installed/initialized the default meter before using it.

Reply:
What about creating the global `Meter` lazily? That would still let users configure `Meter` at runtime as long as they do so before the first call. I think it's a worthwhile goal to have a working default `Tracer` and `Meter` out of the box, without any user-provided config.

Reply:
When you say lazily, does that mean instantiating a `Meter` when the API is imported? Similarly for `tracer`:
```
_TRACER = None  # type: typing.Optional[Tracer]
_TRACER_FACTORY = None  # type: typing.Optional[ImplementationFactory]
```
would be 
```
_TRACER = Tracer() # type: typing.Optional[Tracer]
_TRACER_FACTORY = None  # type: typing.Optional[ImplementationFactory]
```
Something like this?